<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VFormLogin</class>
 <widget class="QWidget" name="VFormLogin">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>740</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>STEMe-Đăng nhập</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../image.qrc">
    <normaloff>:/image/Steme_icon.jpg</normaloff>:/image/Steme_icon.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #132635;
background-color: #193848;
</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>119</y>
     <width>460</width>
     <height>491</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #193848;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="pushButton_Login">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>400</y>
      <width>300</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border: none;
background-color: rgb(0, 222, 99);
	color: rgb(0, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>Đăng nhập</string>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>20</height>
     </size>
    </property>
    <property name="shortcut">
     <string>Enter</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>78</x>
      <y>246</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(13, 207, 200);</string>
    </property>
    <property name="text">
     <string>Mật khẩu</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Password">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>270</y>
      <width>300</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    border: 1px solid gray;
	color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string extracomment="Mật khẩu"/>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>370</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Loại tài khoản:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_accountType">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>370</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255,255, 255);</string>
    </property>
    <property name="text">
     <string>Student (học sinh)</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>361</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #193848;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>29</x>
       <y>15</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(13, 207, 200);</string>
     </property>
     <property name="text">
      <string>Tên đăng nhập</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_Account">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>300</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox {
    border: 1px solid gray;
	color: rgb(255, 255, 255);
}
QComboBox::drop-down {
    width:35px;
}
</string>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>79</x>
      <y>305</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(13, 207, 200);</string>
    </property>
    <property name="text">
     <string>Cổng kết nối</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_Port">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>330</y>
      <width>301</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox {
    border: 1px solid gray;
	color: rgb(255, 255, 255);
}
QComboBox::drop-down {
    width:35px;
}</string>
    </property>
   </widget>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>488</x>
     <y>140</y>
     <width>300</width>
     <height>150</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/image/LogginLogoBig.png);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>comboBox_Account</tabstop>
  <tabstop>lineEdit_Password</tabstop>
  <tabstop>pushButton_Login</tabstop>
  <tabstop>graphicsView</tabstop>
 </tabstops>
 <resources>
  <include location="../image.qrc"/>
 </resources>
 <connections/>
</ui>
